shopinvader_search_engine/models/shopinvader_backend.py

```diff
@@ -29,3 +39,22 @@ def clear_index(self):
         for index in self.mapped('se_backend_id.index_ids'):
             index.clear_index()
         return True
+
+    @api.multi
+    def add_misssing_index(self):
+        self.ensure_one()
+        models = self._get_default_model()
+        index_obj = self.env['se.index']
+        ir_export_obj = self.env['ir.exports']
+        for model in models:
+            for lang in self.lang_ids:
+                if not self.index_ids.filtered(
+                        lambda i: i.lang_id == lang and i.model_id == model):
+                    ir_export = ir_export_obj.search([('resource', '=', model.model)])
```

Comment:
@mourad-ehm More than one 'ir.export' can be defined for same model.... `limit=1`